### DISCUSSION TITLE
Is anyone using this for SSR of a javascript component

### DISCUSSION THREAD

**Discussion by runxc1** [microsoft/node-api-dotnet#385] [General]
I've got a dotnet application that uses vuejs for a lot of the front end.  It's heavily cached and currently does all the rendering on the client side but in some scenarios parts of the application would benefit from server side rendering and I'm wondering if anyone has used this project to build something like that.    I'd be interested in examples in vuejs, react or svelte if anyone has gone down that road.   Seems it'd be helpful for SEO.

**Comment by schmitch** [comment]
actually my company uses reactjs.net and we are looking forward to use node-api-dotnet. I stumblued upon the project in the last 2 weeks and build some prototypish code - which I can’t release yet, but it works extremly well and it even supports readable stream, so suspense is working. still needs a builder and good glue code from the js side especially for routing.

**Comment by wasabii** [comment]
I am working on a resurrected version of NodeServices using this.
